Don't quit just because we don't have a good Zaptel timing source.
See http://bugs.debian.org/491310

 -- Tzafrir Cohen <tzafrir.cohen@xorcom.com>

--- a/main/asterisk.c
+++ b/main/asterisk.c
@@ -3288,9 +3288,8 @@ int main(int argc, char *argv[])
 					"\n\t2. You only have to load DAHDI drivers if you want to take advantage of DAHDI services.  One option is to unload DAHDI modules if you don't need them."
 					"\n\t3. If you need DAHDI services, you must correctly configure DAHDI."
 				};
-				ast_log(LOG_ERROR, "%s\n", dahdi_timer_error);
-				usleep(100);
-				exit(1);
+				ast_log(LOG_WARNING, "%s\n", dahdi_timer_error);
+				ast_log(LOG_WARNING, "However, on Debian we don't quit that easily. See http://bugs.debian.org/491310\n");
 			}
 			close(fd);
 		}
